### ISSUE TITLE
3.2 Crash On Load

### ISSUE THREAD

**Issue by nicholasnelson** [Tempus/Spire-with-Friends#58]
Since the update on 2022/10/05, Spire With Friends causes the game to crash when loading.

```
Version Info:
 - Java version (1.8.0_144)
 - Slay the Spire (10-04-2022)
 - ModTheSpire (3.29.0)
Mod list:
 - basemod (5.45.0)
 - chronoMods (3.2)

Begin patching...
Patching enums...Done.
Finding core patches...
Finding patches...
Patching Overrides...
Injecting patches...Done.
Compiling patched classes...Done.
Busting enums...Done.

Setting isModded = true...Done.

Adding ModTheSpire to version...Done.

Initializing mods...
 - BaseMod
   - basemod.BaseMod
libgdx version 1.9.5
06:07:09.921 ERROR basemod.DevConsole> Failed to load dev console history: java.nio.file.NoSuchFileException: C:\Users\snip\AppData\Local\ModTheSpire\BaseMod\console-history.txt
   - 475ms
 - Spire with Friends
   - chronoMods.TogetherManager
   - 3ms
Done.

Starting game...
Exception in thread "LWJGL Application" com.badlogic.gdx.utils.GdxRuntimeException: java.lang.NoClassDefFoundError: com/codedisaster/steamworks/SteamNetworking$API
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:133)
Caused by: java.lang.NoClassDefFoundError: com/codedisaster/steamworks/SteamNetworking$API
	at chronoMods.network.steam.SteamIntegration.initialize(SteamIntegration.java:68)
	at chronoMods.network.NetworkHelper.initialize(NetworkHelper.java:80)
	at chronoMods.TogetherManager.receivePostInitialize(TogetherManager.java:274)
	at basemod.BaseMod.publishPostInitialize(BaseMod.java:2260)
	at basemod.patches.com.megacrit.cardcrawl.core.CardCrawlGame.PostInitializeHook.Postfix(PostInitializeHook.java:9)
	at com.megacrit.cardcrawl.core.CardCrawlGame.create(CardCrawlGame.java:307)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication.mainLoop(LwjglApplication.java:149)
	at com.badlogic.gdx.backends.lwjgl.LwjglApplication$1.run(LwjglApplication.java:126)
Caused by: java.lang.ClassNotFoundException: com.codedisaster.steamworks.SteamNetworking$API
	at com.evacipated.cardcrawl.modthespire.MTSClassLoader.findClass(MTSClassLoader.java:112)
	at com.evacipated.cardcrawl.modthespire.MTSClassLoader.loadClass(MTSClassLoader.java:86)
	... 8 more
```

Reverting the changes to SteamIntegration and SteamPlayer from [02e2124](https://github.com/Tempus/Spire-with-Friends/commit/02e21245e1e77ec0e55b9a8eb273765497d453fa) fixed it for me.

**Comment by ngokevin** [comment]
We had two people on MacOS and two people on Windows with the latest 3.2

- People on MacOS loaded successfully
- People on Windows failed with that error

**Comment by nicholasnelson** [comment]
Fixed by #59
